autoload/neomake/makers/ft/elixir.vim

```diff
@@ -1,12 +1,28 @@
 " vim: ts=4 sw=4 et
 
+let g:neomake_elixir_credo_type_map= get(g:, 'neomake_elixir_credo_type_map', {'F': 'W','C': 'W','D': 'I','R': 'I'})
```

Comment:
Please use `s:` for the default, and then use it with `get()` below - it should not set this var into the global namespace.